orchestration/dags/data_gcp_dbt/profiles.yml

```diff
@@ -72,3 +72,37 @@ elementary:
       project: passculture-data-prod
       location: europe-west1
       threads: 2
+
+CI:
```

Comment:
Si le dataset CI_ a un temps de vie d'1 jour et qu'on lance uniquement les modèles modifiés, je pense qu'il risque d'y avoir bcp d'erreur de compilation avec des tables not found ?